code/src/index.css

```diff
@@ -1,13 +1,125 @@
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  display: flex;
+  justify-content: space-evenly;
+  font-family: "Inconsolata";
+  background-image: url(assets/cork-background.jpg);
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.todo-container {
+  height: 430px;
+  background-color: rgb(247, 230, 197);
+  border-radius: 25px;
+  padding: 16px 35px 16px 35px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 25px;
+  position: relative;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+}
+
+.header-text {
+  font-size: 18px;
+  font-weight: 700;
+}
+
+.todo-item {
+  display: flex;
+  margin: 12px;
+}
+
+.todo-text {
+  font-size: 17px;
+  margin: 7px 10px 7px 10px;
+}
+
+.todo-text-container {
+  display: flex;
+  flex-direction: column;
+  align-items: baseline;
+}
+
+.todo-item button {
+  position: absolute;
+  right: 20px;
+}
+
+.text-input {
+  font-size: 17px;
+  border-radius: 5px;
+  height: 30px;
+  margin-right: 5px;
+  font-family: "Inconsolata";
+  background-color: whitesmoke;
+  padding: 5px;
+  text-align: center;
+  margin-bottom: 10px;
+}
+
+.date {
+  font-size: 9px;
+  display: flex;
+  justify-content: space-around;
+}
+
+/* ------ checkbox ------ */
+
+[type="checkbox"]:not(:checked) {
+  background-color: whitesmoke;
+  width: 35px;
+  height: 35px;
+  position: absolute;
+  left: 20px;
+  cursor: pointer;
+  opacity: 1;
+  transform: scale(1);
+  transition: all 0.3s ease;
+}
+
+[type="checkbox"]:checked {
+  background-color: whitesmoke;
+  width: 30px;
+  height: 30px;
+  background-size: contain;
+  opacity: 1;
+  transform: scale(1);
+  transition: all 0.3s ease;
+  position: absolute;
+  left: 20px;
+  cursor: pointer;
+}
+
+/* -----TASKCOUNTER ------- */
+
+.taskCounter-container {
+  position: absolute;
+  bottom: 25px;
+  display: flex;
+  align-items: center;
+  /* border-top: 2px solid rgb(154, 153, 153); */
+}
+
+/*-------------------------------------*/
+/* ----- TABLET AND LAPTOP VIEW ----- */
+/*-----------------------------------*/
+
+@media (min-width: 667px) {
+  .todo-container {
+    width: 100%;
+  }
+
+  .todo-item button {
+    position: absolute;
+    right: 35px;
+  }
+
+  [type="checkbox"]:not(:checked) {
+    position: absolute;
+    left: 35px;
+  }
+
+  [type="checkbox"]:checked {
+    position: absolute;
+    left: 35px;
+  }
```

Comment:
You don't need the `positon:absolute `here in the media queries, you already have them in the original part of the code :)